bgzf.c

```diff
@@ -182,6 +210,7 @@ BGZF *bgzf_open(const char *path, const char *mode)
         hFILE *fpw;
         if ((fpw = hopen(path, mode)) == 0) return 0;
```

Comment:
For readability could we make these == NULL and return NULL too? Makes it easier to read.